lisa/secret.py

```diff
@@ -69,3 +75,20 @@ def mask(input: str) -> str:
         if secret[0] in input:
             input = input.replace(secret[0], secret[1])
     return input
+
+
+def add_secret_for_sasurl(value: str) -> None:
+    sasurl_match = PATTERN_SASURL[0].findall(value)
+    if sasurl_match:
+        add_secret(value, PATTERN_SASURL)
+
+
+def check_secret(obj: Any) -> None:
```

Comment:
This method is not necessary. The log will be filtered automatically.

Reply:
I changed the method name to check_sasurl. In this method, it checks if there is a value that has sasurl pattern, if so, then add secret for this value.